libs/velo-external-db-types/src/index.ts

```diff
@@ -140,16 +141,59 @@ export type InputField = FieldAttributes & { name: string }
 
 export type ResponseField = FieldAttributes & { field: string }
 
+export type ColumnCapabilities = {
```

Comment:
You asked me to move all the index types to another file, maybe it would be good idea here as well?